src/views/Project/index.jsx

```diff
@@ -0,0 +1,247 @@
+import { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import ReactMarkdown from 'react-markdown';
+import { ApolloConsumer } from 'react-apollo';
+import _ from 'lodash';
+import gql from 'graphql-tag';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+import Issues from './issues.graphql';
+import Bugs from './bugs.graphql';
+
+class Project extends Component {
+  constructor(props) {
+    super(props);
+    const fileName = props.match.params.projectKey;
```

Comment:
I think we should move this logic to `static getDerivedStateFromProps`.

https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops